open_rarity/resolver/opensea_api_helpers.py

```diff
@@ -86,34 +89,69 @@ def fetch_opensea_assets_data(
 
 
     """
+    client = kwargs.get("client", None)
     assert len(token_ids) <= limit
     # Max 30 limit enforced on API
     assert limit <= 30
     querystring = {
-        "token_ids": token_ids,
+        "token_ids": [int(i) for i in token_ids],
```

Comment:
is the input not always a list of integers?

Reply:
So yes you would think... It is actually a side effect of the `array_split` which converts to `np.int64` and then the `httpx` library incorrectly parses the query params and makes the url look like `/v1/assets?token_ids=%5B5040+5041+5042+5043+5044+5045`. I will fix a different way.